.github/workflows/commit-lint.yml

```
@@ -0,0 +1,30 @@
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+
+      - name: Install dependencies
+        run: npm ci --legacy-peer-deps
+
+      - name: Run commitlint on PR
+        run: |
+          npx commitlint --from ${{ github.event.pull_request.base.sha }} --to ${{ github.event.pull_request.head.sha }} --verbose 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)